From ee52dae3a707cb7f11265a478f3364424e1aa4f6 Mon Sep 17 00:00:00 2001
From: Jason Monk <jmonk@google.com>
Date: Thu, 12 Apr 2018 14:58:06 +0000
Subject: [PATCH 184/306] Revert "Clear out tuner prefs since it is disabled"

Bug: 74851117
Fixes: 77937472
This reverts commit 8bb9bbf152e35cdb4263aa1132827d51eb57e8b4.

Change-Id: I2dea3fec7ca3f59c63ce9395ba2784ca9dc58316
(cherry picked from commit a3e5552c475c328fcfb385ab4bdbb34cb556f5fb)
---
 .../src/com/android/systemui/tuner/TunerServiceImpl.java    | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/tuner/TunerServiceImpl.java b/packages/SystemUI/src/com/android/systemui/tuner/TunerServiceImpl.java
index 639e49b1a17..5a4478f072e 100644
--- a/packages/SystemUI/src/com/android/systemui/tuner/TunerServiceImpl.java
+++ b/packages/SystemUI/src/com/android/systemui/tuner/TunerServiceImpl.java
@@ -58,7 +58,7 @@ public class TunerServiceImpl extends TunerService {
 
     private static final String TUNER_VERSION = "sysui_tuner_version";
 
-    private static final int CURRENT_TUNER_VERSION = 3;
+    private static final int CURRENT_TUNER_VERSION = 2;
 
     private final Observer mObserver = new Observer();
     // Map of Uris we listen on to their settings keys.
@@ -119,10 +119,6 @@ public class TunerServiceImpl extends TunerService {
         if (oldVersion < 2) {
             setTunerEnabled(mContext, false);
         }
-        if (oldVersion < 3) {
-            // Delay this so that we can wait for everything to be registered first.
-            new Handler(Dependency.get(Dependency.BG_LOOPER)).postDelayed(() -> clearAll(), 5000);
-        }
         setValue(TUNER_VERSION, newVersion);
     }
 
-- 
2.17.1

